libraries/rodeos/include/b1/rodeos/rodeos_tables.hpp

```diff
@@ -225,11 +225,13 @@ void store_delta_kv(eosio::kv_environment environment, table_delta_v0& delta, F
       f();
       auto  obj  = eosio::from_bin<key_value>(row.data);
       auto& obj0 = std::get<key_value_v0>(obj);
+      /* TODO: uncomment this
```

Comment:
It was doing something before, so don't remove it without explaining to us why it can be removed.

Reply:
It got affected by the abieos change in SHiP protocol to the key value, because rodeos is currently not working (we need to remove the database parameter and make more modifications) i left this comment to address at that time. I added a longer comment explaining this.